src/main/java/com.amazon.bookstore/BookstoreController.java

```diff
@@ -52,12 +52,6 @@ public String mainpage(Model model) {
         return "ownerBookstore";
     }
 
-    @PostMapping("/editBookstore")
```

Comment:
You're still calling /editBookstore in the ownerBookstore.html so removing this causes an error

Reply:
fixed!

Reply:
added it back, Lynn will be using this.